test/functional/p2p_node_network_limited.py

```diff
@@ -4,9 +4,9 @@
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """Tests NODE_NETWORK_LIMITED.
 
-Tests that a node configured with -prune=550 signals NODE_NETWORK_LIMITED correctly
+Tests that a node configured with -prune=2250 signals NODE_NETWORK_LIMITED correctly
```

Comment:
Should this be testing -prune=2200 instead of -prune=2250 to be more consistent with the original test? (also down below on line 38)

Reply:
I think you're right, I'll dig into that.